docs/tutorial/using-backup-with-syclladb.md

```diff
@@ -0,0 +1,235 @@
+# Using Backup with SycllaDB
+
+This article will show you how to deploy Vald with ScyallaDB as backup Database using Helm and run it on your kubernetes cluster.
+
+## Overview
+
+One of the Vald's feature, Vald has auto index backup feature using MySQL + Redis or Cassandra which enables disaster recovery.
+In this tutorial, you will use ScyllaDB for backup Persistent Volume and deploy more microsevices than [Get Started](../tutorial/get-started.md).
```

Comment:
```suggestion
In this tutorial, you will use ScyllaDB for backup to Persistent Volume.<br>
It will deploy more Vald component than [Get Started](../tutorial/get-started.md).
```

Reply:
refer